### PR TITLE
[Merged by Bors] - chore: update fluvio to 0.10.8

### PR DESCRIPTION
**Pull request by nacardin**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -20,9 +20,12 @@ serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
 node-bindgen = "5.1.0"
 flate2 = "1.0.24"
-fluvio = { version = "0.18.1", features = ["admin"] }
+fluvio = { version = "0.18.2", features = ["admin"] }
```
Comment (morenol) [inline Cargo.toml:23]: I think that we should move from using crates.io to github tags here, like how we do in connectors

File: Cargo.toml [hunk 2]
```diff
@@ -20,7 +20,7 @@ serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
 node-bindgen = "5.1.0"
 flate2 = "1.0.24"
-fluvio = { version = "0.18.1", features = ["admin"] }
+fluvio = { features = ["admin"], git = "https://github.com/nacardin/fluvio.git", rev = "434a089e55826fe1f034eb5a4e061630afe8733d" }
```
Comment (morenol) [inline Cargo.toml:23]: maybe the tag instead?

```suggestion
fluvio = { features = ["admin"], git = "https://github.com/nacardin/fluvio.git", tag = "v0.10.8-dev-1" }
```

Comment (nacardin) [reply Cargo.toml:23]: there was a bug in `v0.10.8-dev-1`, will update to `v0.10.8-dev-2` when ready

File: Cargo.toml [hunk 3]
```diff
@@ -20,7 +20,7 @@ serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
 node-bindgen = "5.1.0"
 flate2 = "1.0.24"
-fluvio = { version = "0.18.1", features = ["admin"] }
+fluvio = { features = ["admin"], git = "https://github.com/infinyon/fluvio.git", tag = "v0.10.8" }
 fluvio-future = { version = "0.5.1", features = ["tls", "task", "io"] }
 fluvio-spu-schema = "0.13.1"
```
Comment (morenol) [inline Cargo.toml:25]:
```suggestion
fluvio-spu-schema = { git = "https://github.com/infinyon/fluvio.git", tag = "v0.10.8" } 
```

**Review by morenol — APPROVED** [review]

**Pull request closed without merging**